tools/base_labels/service.py

```
@@ -0,0 +1,228 @@
+from . import config as cfg
+from .parser import extract_display_names
+from .translation import call_label_translation_model, default_model
+
+
+def load_config(path: Path | None = None) -> dict[str, Any]:
+    path = path or cfg.CONFIG_PATH
+    if not path.is_file():
+        return {"version": 1, "bases": {}}
+    data = json.loads(path.read_text(encoding="utf-8-sig"))
+    if not isinstance(data, dict):
+        raise ValueError("base_display_names.json must contain a JSON object")
```

Ruff (0.15.15)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)